devlab/tests/functional_test.py

```diff
@@ -182,3 +183,10 @@ def get_vms_with_fip_associated(self):
         vms = config.vms
         [vms.extend(i['vms']) for i in config.tenants if 'vms' in i]
         return [vm['name'] for vm in vms if vm.get('fip')]
+
+    def is_tenant_exist(self, keystone_client, tenant_id):
```

Comment:
The problem is that we don't have such method for tenant_id, and here we can't get tenant_name for vms we've got from SRC cloud. So, i'll prefer to keep that method as well as method in generate_load.